### DISCUSSION TITLE
read_era5 2D spectra produce different Hs to bulk paramter

### DISCUSSION THREAD

**Discussion by Chaminu** [wavespectra/wavespectra#127] [General]
I have downloaded the ERA5 2D-spectral data in netcdf format and read it using [wavespectra.io],
The 1D and 2D spectra look good. However when I calculate  Hs (Hmo) and compare it to the bulk parameters (SWH) obtained from the ERA5 meteorological download, I end up with a 10% difference. I have attached a comparison during June 2023.  Any help would be appreciated.
![ERA5_wave_Hmo_swh](https://github.com/wavespectra/wavespectra/assets/153249329/a922ffa0-c8bb-4e4c-b924-5b99c8a85ad9)

**Comment by rafa-guedes** [comment]
Hi @Chaminu 
 
the era5 reader was written by @JohnCHarrington, perhaps he could have some ideas on whether there could be something special about the way the ERA5 spectra are stored or SWH is calculated. For example, perhaps not the entire frequency range could be saved in the spectra. It is interesting as the relative offset looks relatively constant but during some periods the two values are nearly the same. I think ERA5 provides bulk significant wave height values from the raw model output but also satellite-corrected values so this could be another thing to pay attention to.

**Comment by Chaminu** [comment]
Hi @rafa-guedes 
I know WISWAVE wave model stored the tail of the spectrum outside of the 2D spectra, but thIs was a 2nd generation model (ie wave/wave interaction was parameterized).   I see no variable offered in the ERA5 output that may be a 'tail'. 
I will look further into your suggestion on satellite-correct values. 
It would be good for someone to duplicate the issue before I approach ECMWF, also If anyone has an ERA5 contact it would be appreciated.
Thanks

**Comment by Chaminu** [comment]
<html><body>
<div>I check on the Satellite issue, No problem there:</div>
"Altimeter corrected wave height" ShortName: acwh, or  parameter: 247.140
<div>
I am downloading:" Significant  height of combined wind waves and swell" Shortname: swh. or parameter: 229.140 </div> For the 2D spectral I extract:'param': '251.140',
Regards

</body>
</html>
